### ISSUE TITLE
New submissions for Mon, 13 Apr 20

### ISSUE THREAD

**Issue by kobiso** [kobiso/daily-arxiv-noti#62]
## Keyword: metric learning
There is no result 
## Keyword: image retrieval
There is no result 
### Beyond Disentangled Representations: An Attentive Angular Distillation  Approach to Large-scale Lightweight Age-Invariant Face Recognition
 - **Authors:** Thanh-Dat Truong, Chi Nhan Duong, Kha Gia Quach, Dung Nguyen, Ngan Le, Khoa Luu, Tien D. Bui
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2004.05085
 - **Pdf link:** https://arxiv.org/pdf/2004.05085
 - **Abstract**
 Disentangled representations have been commonly adopted to Age-invariant Face Recognition (AiFR) tasks. However, these methods have reached some limitations with (1) the requirement of large-scale face recognition (FR) training data with age labels, which is limited in practice; (2) heavy deep network architecture for high performance; and (3) their evaluations are usually taken place on age-related face databases while neglecting the standard large-scale FR databases to guarantee its robustness. This work presents a novel Attentive Angular Distillation (AAD) approach to Large-scale Lightweight AiFR that overcomes these limitations. Given two high-performance heavy networks as teachers with different specialized knowledge, AAD introduces a learning paradigm to efficiently distill the age-invariant attentive and angular knowledge from those teachers to a lightweight student network making it more powerful with higher FR accuracy and robust against age factor. Consequently, AAD approach is able to take the advantages of both FR datasets with and without age labels to train an AiFR model. Far apart from prior distillation methods mainly focusing on accuracy and compression ratios in closed-set problems, our AAD aims to solve the open-set problem, i.e. large-scale face recognition. Evaluations on LFW, IJB-B and IJB-C Janus, AgeDB and MegaFace-FGNet with one million distractors have demonstrated the efficiency of the proposed approach. This work also presents a new longitudinal face aging (LogiFace) database for further studies in age-related facial problems in future.
### SMART Paths for Latency Reduction in ReRAM Processing-In-Memory  Architecture for CNN Inference
 - **Authors:** Sho Ko, Shimeng Yu
 - **Subjects:** Hardware Architecture (cs.AR)
 - **Arxiv link:** https://arxiv.org/abs/2004.04865
 - **Pdf link:** https://arxiv.org/pdf/2004.04865
 - **Abstract**
 This research work proposes a design of an analog ReRAM-based PIM (processing-in-memory) architecture for fast and efficient CNN (convolutional neural network) inference. For the overall architecture, we use the basic hardware hierarchy such as node, tile, core, and subarray. On the top of that, we design intra-layer pipelining, inter-layer pipelining, and batch pipelining to exploit parallelism in the architecture and increase overall throughput for the inference of an input image stream. We also optimize the performance of the NoC (network-on-chip) routers by decreasing hop counts using SMART (single-cycle multi-hop asynchronous repeated traversal) flow control. Finally, we experiment with weight replications for different CNN layers in VGG (A-E) for large-scale data set ImageNet. In our simulation, we achieve 40.4027 TOPS (tera-operations per second) for the best-case performance, which corresponds to over 1029 FPS (frames per second). We also achieve 3.5914 TOPS/W (tera-operaions per second per watt) for the best-case energy efficiency. In addition, the architecture with aggressive pipelining and weight replications can achieve 14X speedup compared to the baseline architecture with basic pipelining, and SMART flow control achieves 1.08X speedup in this architecture compared to the baseline. Last but not least, we also evaluate the performance of SMART flow control using synthetic traffic.
### Improved Residual Networks for Image and Video Recognition
 - **Authors:** Ionut Cosmin Duta, Li Liu, Fan Zhu, Ling Shao
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2004.04989
 - **Pdf link:** https://arxiv.org/pdf/2004.04989
 - **Abstract**
 Residual networks (ResNets) represent a powerful type of convolutional neural network (CNN) architecture, widely adopted and used in various tasks. In this work we propose an improved version of ResNets. Our proposed improvements address all three main components of a ResNet: the flow of information through the network layers, the residual building block, and the projection shortcut. We are able to show consistent improvements in accuracy and learning convergence over the baseline. For instance, on ImageNet dataset, using the ResNet with 50 layers, for top-1 accuracy we can report a 1.19% improvement over the baseline in one setting and around 2% boost in another. Importantly, these improvements are obtained without increasing the model complexity. Our proposed approach allows us to train extremely deep networks, while the baseline shows severe optimization issues. We report results on three tasks over six datasets: image classification (ImageNet, CIFAR-10 and CIFAR-100), object detection (COCO) and video action recognition (Kinetics-400 and Something-Something-v2). In the deep learning era, we establish a new milestone for the depth of a CNN. We successfully train a 404-layer deep CNN on the ImageNet dataset and a 3002-layer network on CIFAR-10 and CIFAR-100, while the baseline is not able to converge at such extreme depths. Code is available at: https://github.com/iduta/iresnet
### Exemplar VAEs for Exemplar based Generation and Data Augmentation
 - **Authors:** Sajad Norouzi, David J. Fleet, Mohammad Norouzi
 - **Subjects:** Machine Learning (cs.LG); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2004.04795
 - **Pdf link:** https://arxiv.org/pdf/2004.04795
 - **Abstract**
 This paper presents a framework for exemplar based generative modeling, featuring Exemplar VAEs. To generate a sample from the Exemplar VAE, one first draws a random exemplar from a training dataset, and then stochastically transforms that exemplar into a latent code, which is then used to generate a new observation. We show that the Exemplar VAE can be interpreted as a VAE with a mixture of Gaussians prior in the latent space, with Gaussian means defined by the latent encoding of the exemplars. To enable optimization and avoid overfitting, Exemplar VAE's parameters are learned using leave-one-out and exemplar subsampling, where, for the generation of each data point, we build a prior based on a random subset of the remaining data points. To accelerate learning, which requires finding the exemplars that exert the greatest influence on the generation of each data point, we use approximate nearest neighbor search in the latent space, yielding a lower bound on the log marginal likelihood. Experiments demonstrate the effectiveness of Exemplar VAEs in density estimation, representation learning, and generative data augmentation for supervised learning.
### Analysis on DeepLabV3+ Performance for Automatic Steel Defects Detection
 - **Authors:** Zheng Nie, Jiachen Xu, Shengchang Zhang
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2004.04822
 - **Pdf link:** https://arxiv.org/pdf/2004.04822
 - **Abstract**
 Our works experimented DeepLabV3+ with different backbones on a large volume of steel images aiming to automatically detect different types of steel defects. Our methods applied random weighted augmentation to balance different defects types in the training set. And then applied DeeplabV3+ model three different backbones, ResNet, DenseNet and EfficientNet, on segmenting defection regions on the steel images. Based on experiments, we found that applying ResNet101 or EfficientNet as backbones could reach the best IoU scores on the test set, which is around 0.57, comparing with 0.325 for using DenseNet. Also, DeepLabV3+ model with ResNet101 as backbone has the fewest training time.
### Fully Automatic Electrocardiogram Classification System based on  Generative Adversarial Network with Auxiliary Classifier
 - **Authors:** Zhanhong Zhou, Xiaolong Zhai, Chung Tin
 - **Subjects:** Machine Learning (cs.LG); Signal Processing (eess.SP); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2004.04894
 - **Pdf link:** https://arxiv.org/pdf/2004.04894
 - **Abstract**
 A generative adversarial network (GAN) based fully automatic electrocardiogram (ECG) arrhythmia classification system with high performance is presented in this paper. We have designed a discriminator (D) to take ECG coupling matrix as input, and then predict the input validity (real or generated) as well as arrhythmia classes. The generator (G) in our GAN is designed to generate various coupling matrix inputs conditioned on different arrhythmia classes for data augmentation. Upon completion of training for our GAN, we extracted the trained D as an arrhythmia classifier in a transfer learning manner. After fine-tuning D by including patient-specific normal beats estimated using an unsupervised algorithm, and generated abnormal beats by G that are usually rare to obtain, our fully automatic system showed superior overall classification performance for both supraventricular ectopic beats (SVEB or S beats) and ventricular ectopic beats (VEB or V beats) on the MIT-BIH arrhythmia database. It surpassed several state-of-art automatic classifiers and can perform on similar levels as some expert-assisted methods. In particular, the F1 score of SVEB has been improved by up to 12% over the top-performing automatic systems. Moreover, high sensitivity for both SVEB (85%) and VEB (93%) detection has been achieved, which is of great value for practical diagnosis. We, therefore, suggest our ACE-GAN (Generative Adversarial Network with Auxiliary Classifier for Electrocardiogram) based automatic system can be a promising and reliable tool for high throughput clinical screening practice, without any need of manual intervene or expert assisted labeling.
### MA 3 : Model Agnostic Adversarial Augmentation for Few Shot learning
 - **Authors:** Rohit Jena, Shirsendu Sukanta Halder, Katia Sycara
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2004.05100
 - **Pdf link:** https://arxiv.org/pdf/2004.05100
 - **Abstract**
 Despite the recent developments in vision-related problems using deep neural networks, there still remains a wide scope in the improvement of generalizing these models to unseen examples. In this paper, we explore the domain of few-shot learning with a novel augmentation technique. In contrast to other generative augmentation techniques, where the distribution over input images are learnt, we propose to learn the probability distribution over the image transformation parameters which are easier and quicker to learn. Our technique is fully differentiable which enables its extension to versatile data-sets and base models. We evaluate our proposed method on multiple base-networks and 2 data-sets to establish the robustness and efficiency of this method. We obtain an improvement of nearly 4% by adding our augmentation module without making any change in network architectures. We also make the code readily available for usage by the community.
### Brain Interface Based Wheel Chair Control System for Handicap -- An  advance and viable approach
 - **Authors:** Mohtashim Baqar, Azfar Ghani, Azeem Aftab, Shahzad Karim Khawar
 - **Subjects:** Human-Computer Interaction (cs.HC)
 - **Arxiv link:** https://arxiv.org/abs/2004.04860
 - **Pdf link:** https://arxiv.org/pdf/2004.04860
 - **Abstract**
 This paper presents advancement towards making an efficient and viable wheel chair control system based on brain computer interface via electro-oculogram (EOG) signals. The system utilizes the movement of eye as the element of purpose for controlling the movement of the wheel chair. Skin-surface electrodes are placed over skin for the purpose of acquiring the electro-oculogram signal and with the help of differential amplifier the bio-potential is measured between the reference and the point of interest, afterwards these obtained low voltage pulses are amplified, then passed through a sallen-key filter for noise removal and smoothening. These pulses are then collected on to the micro-controller; based on these pulses motor is switched to move in either right or left direction. A prototype system was developed and tested. The system showed promising results. The test conducted showed 99.5% efficiency of movement in correct direction.
## Keyword: try-on
There is no result
